apps/website/src/content/docs/button.mdx

```diff
@@ -93,7 +93,7 @@ Pressing this button will always open a menu. Upon selecting an option from the
 
 <PropsTable path={frontmatter.propsPathDropdownButton} />
 
-## Icon button
+## IconButton
```

Comment:
i changed this here because it affects the fragment link that i'm using in `/tooltip`.

we should get rid of these spaces everywhere on all pages, and use the actual component name instead.